src/adapter.rs

```diff
@@ -169,21 +136,18 @@ impl Adapter for SqlxAdapter {
     }
 
     async fn load_filtered_policy<'a>(&mut self, m: &mut dyn Model, f: Filter<'a>) -> Result<()> {
-        let rules = adapter::load_policy(&self.pool).await?;
+        let rules = adapter::load_filtered_policy(&self.pool, &f).await?;
+        self.is_filtered = true;
```

Comment:
It'll be good if this can show the real status, `is_filtered` or not, let's say:

```
n_fetched < n_total
```

Otherwise it's good, thanks for your contribution

Reply:
How do you propose getting `n_total`?

The idea here is that it is a performance optimisation by filtering on the database. Getting `n_total` to compare implies making another database call degrading the performance.
Isn't it intuitive for the caller to assume if they call a `filtered` method that it will be `filtered`? That it actually isn't filtered is an edge case that no caller should ever rely on - or?

Reply:
Yes, I understand it can be tricky...

Let's merge it first